meetings/2017-08-10.md

```diff
@@ -0,0 +1,94 @@
+[x] JoshBW: will evaluate HackerOne and BugCrowd (https://github.com/nodejs/security-wg/issues/16, https://github.com/nodejs/security-wg/issues/17)
+[x] JoshBW: will investigate what it takes to get a semmle.com license for Node.js as an open source project (https://github.com/nodejs/security-wg/issues/29)
+
+# Agenda
+
+## HackerOne Demo
+
+Reed gave a demo of HackerOne’s dashboard.  He sent out a bunch of demo
+accounts.  If you didn’t get an email and would like a demo account please ping
+@reedloden (reed @ hackerone.com)
+
+Can’t set an “auto-disclosure” after X days of filings in the settings (though
+you can after a bug was resolved), but probably possible with the API.
+Normally disclosure requires an intentional act within the report UI.
+
+Full API (https://api.hackerone.com).  Probably not a Node.js library for it
+yet (Ruby, Python, Go, etc. are present)
+
+Can issue CVEs for both node and npmjs.org packages if we choose to go down
+that route.
+
+Two possible use-cases:
+1. Node vulnerabilities
+2. Npmjs.org package vulnerabilities
+
+For npmjs.org packages, HackerOne looks to have a great feature set, CVE
+issuance and the rich permission model among them. I’d like to see some
+investigation of the API to ensure we can pull issues out as JSON for storage
+in github.com, but otherwise I think we should try it out.
+
+For node vulnerabilities, we’ll have to pitch HackerOne to the core members of
+the Node Security Response Team, Ben and Fyodor at least. The thing they might
+find most compelling for use with node core relates to permissions, since the
+Response team size has been described as too large recently. Use of HackerOne
+will allow:
+- Web or email submission of vulnerabilities, with the submitter being allowed
+  to view and discuss the submission directly using the web UI.
+- The conversation history to be made public when a vulnerability is
+  publicized: ATM all conversations about vulnerabilities are private for ever,
+  unnecessarily.
+- A triage team of 3 or 4 people who can see every submission, and either
+  reject it as not a vulnerability, or assign it to the correct people to work on
+  it further (those people would get access to only the one specific pre-release
+  vulnerabilty). This would allow, for example, HTTP experts who are not on the
+  Response team to be brought into the discussion of an HTTP issue. It would also
+  allow the setup of a team to further triage npmjs.org package vulnerability
+  reports, so the response team can re-assign non-node core vulnerabilities.
+
+## rubysec organization
+
+- Github: https://github.com/rubysec/ruby-advisory-db
+- Stores vuln per file, covers both gems (`gems/`) and ruby runtimes (`rubies`)
+- Every vuln has a CVE or a OSVDB, but the OSF (open security foundation) has
+  closed shop, and all the OSVDB links are now dead
```

Comment:
https://blog.osvdb.org/2016/04/05/osvdb-fin/ and http://www.securityweek.com/osvdb-shut-down-permanently talk about it somewhat, but the TL;DR is that the majority of the work was being done by Risk Based Security (with very little input from the community), and then others were scraping the site and taking the content for themselves without paying for it or contributing back. So, frankly, just didn't make financial sense for them to keep running the site. :(

This has no impact on CVE allocation at all. One of the big reasons why RubySec used OSVDB for a long time is because it was so hard to get CVEs from MITRE. Nowadays, with more and more companies being CNAs, that isn't a problem anymore. CNAs receive a block of CVEs from MITRE, so they can just assign on-demand.